galsim/phase_screens.py

```diff
@@ -150,6 +176,44 @@ def __hash__(self):
 
     def __ne__(self, other): return not self == other
 
+    def instantiate(self, kmin=0., kmax=float('inf'), check=None):
+        """
+        @param kmin   Minimum k-mode to include when generating phase screens.  Generally this will
+                      only be used when testing the geometric approximation for atmospheric PSFs.
+                      [default: 0]
+        @param kmax   Maximum k-mode to include when generating phase screens.  This may be used in
+                      conjunction with SecondKick to complete the geometric approximation for
+                      atmospheric PSFs.  [default: float('inf')]
```

Comment:
Sure.  I think I started using `float('inf')` instead of `np.inf` because of some problems with the way numpy floats repr vs the way python floats repr in the SED module.  But that doesn't seem to be a problem here, so np.inf is fine.